src/components/Modals/NewCourse.vue

```diff
@@ -42,6 +120,32 @@ export default {
       */
       let currentFocus;
       const inpCopy = inp;
+      function removeActive(x) {
```

Comment:
moved these functions up because es-lint was saying that they were being used before they were declared!

Reply:
Did you edit them at all? Curious for my own PR, this was copied from W3Schools and needs to be changed

Reply:
nope i didnt edit them at all

Reply:
You editing my comment I'm dead